.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -23,13 +23,15 @@ If user exposed functionality or configuration variables are added/changed:
   - [ ] Documentation added/updated in [home-assistant.io](https://github.com/home-assistant/home-assistant.io)
 
 If the code communicates with devices, web services, or third-party tools:
-  - [ ] New dependencies have been added to the `REQUIREMENTS` variable ([example][ex-requir]).
+  - [ ] There is a manifest with all fields filled out correctly ([example][ex-manifest]).
+  - [ ] New dependencies have been added to `requirements` in the manifest ([example][ex-requir]).
```

Comment:
We haven't replaced requirements variable yet, right? We have to update `async_process_deps_reqs` and functions used inside that first.